python/featomic/featomic/clebsch_gordan/_coefficients.py

```diff
@@ -514,10 +518,13 @@ def _cg_couple_dense(
     degree ``o3_lambda``) using CG coefficients. This is a "dense" implementation, using
     all CG coefficients at the same time.
 
-    :param array: input array, we expect a shape of ``[samples, 2*l1 + 1, 2*l2 + 1]``
+    :param array: input array, we expect a shape of ``[samples * properties, 2*l1 + 1,
```

Comment:
should we name the first dimension `N` to make it independent from samples/properties naming?

Reply:
I think this is more opaque, for documentation reasons saying that it is `samples * properties` is useful in my mind

Reply:
The point being that this function does not care about samples/properties. It is only used this way from the outside

Reply:
Ok fair enough - changed!